app/scss/partials/_upgrade_banner.scss

```diff
@@ -19,7 +19,7 @@
 }
 
 .UpgradeBanner--normal {
-	height: 50px;
+	height: 25px;
```

Comment:
Interesting... I had made this change because the height of this banner was preventing a click event on the Tracker Wheel but I can't reproduce the issue.  In fact, it happened on Detailed View but the banner doesn't even use `UpgradeBanner--normal` in that view, it uses `UpgradeBanner--small`.  So this CSS doesn't even effect that.

Sorry for the wild goose chase.

Reverted on `feature/cleanup-cleanup`.

Reply:
The CSS fix looks good. Thanks again.